features/support/stub_feedback_requests.rb

```diff
@@ -1,6 +1,5 @@
 require 'webmock/cucumber'
 
-SURVEY_MONKEY_URL = "https://api.eu.surveymonkey.com/v3/.*"
```

Comment:
As well as this, the other Survey Monkey related constants in this file can go, as can the `Before` blocks that set up survey responses. Also check to see if they are being using in any of the feature files (I don't think they are).

Reply:
Thanks for updating this. I have found a few more in these files:

* `.k8s/live/production` and `.k8s/live/staging` - `SURVEY_MONKEY_COURT_DATA_COLLECTOR_ID`
* `config/settings.` - `survey_monkey_bearer_token`
* `vcr_helper.rb` - `SURVEY_MONKEY_BEARER_TOKEN`

I think that is the lot. A non-case-sensitive global search for `survey` or `monkey` finds these (as well as some unrelated things).

Reply:
Thanks @jrmhaig. I am sorry I didn't spot these the first time.